src/components/BasicHead.vue

```diff
@@ -185,6 +221,14 @@ export default {
 	  this.$store.state.vxg.cmp.BasicHead.show.select = false
 	  this.$store.state.vxg.cmp.BasicHead.show.add = true
 	  this.$store.state.vxg.cmp.BasicHead.show.remove = true
+
```

Comment:
definitely getting to the for loop stage here :)

Reply:
restructured to fit the design